src/dc_mimefactory.rs

```diff
@@ -26,6 +26,8 @@ use crate::location;
 use crate::message::{self, Message};
 use crate::param::*;
 use crate::stock::StockMessage;
+use crate::wrapmime;
+use crate::wrapmime::*;
```

Comment:
duplicate import here, why?

Reply:
i am partly using wrapmime:: instead of directly the imported names.  didn't get around to use it consistently.

Reply:
now using wrapmime::X everywhere, dropped the *-import. there are so many methods and i want to clearly signal with wrapmime::X that it is a safe method, and wraps unsafe mmime stuff.